lib/tasks/backfill_fraud_review_pending_at.rake

```diff
@@ -0,0 +1,70 @@
+namespace :profiles do
+  desc 'If a profile is in GPO and fraud pending state, move it out of fraud pending state'
+
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_fraud_review_pending_at
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_fraud_review_pending_at UPDATE_PROFILES=true
+  #
+  task backfill_fraud_review_pending_at: :environment do |_task, _args|
+    ActiveRecord::Base.connection.execute('SET statement_timeout = 60000')
+
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
+    profiles = Profile.where(
+      'fraud_review_pending_at IS NOT NULL OR fraud_rejection_at IS NOT NULL',
+    ).where.not(
+      gpo_verification_pending_at: nil,
+    )
+
+    profiles.each do |profile|
+      warn "#{profile.id},#{profile.fraud_review_pending_at},#{profile.fraud_rejection_at}"
+      profile.update!(fraud_review_pending_at: nil) if update_profiles
```

Comment:
Should this confirm that fraud_pending_reason is set? And, should it set fraud_rejection_at to nil?

Reply:
Yeah, I'll add a warn if `fraud_pending_reason` is not set and we definitely want to nil `fraud_rejection_at`.